__tests__/app.test.js

```diff
@@ -353,25 +353,55 @@ describe("GET /api/articles (topic query)", () => {
         });
       });
   });
+});
+test("400: returns error when non existant topic is given", () => {
+  return request(app)
+    .get("/api/articles?topic=elephants")
+    .expect(400)
+    .then(({ body }) => {
+      const { msg } = body;
+      expect(msg).toBe("bad request");
+    });
+});
+
+test("200: returns empty array when the topic exists but there are no articles with that topic.", () => {
+  return request(app)
+    .get("/api/articles?topic=paper")
+    .expect(200)
+    .then(({ body }) => {
+      const { articles } = body;
+      expect(articles).toHaveLength(0);
+    });
+});
 
-  test("400: returns error when non existant topic is given", () => {
+describe("GET /api/articles/:article_id (comment_count)", () => {
+  test("200: responds with object with a count of the total of the articles with the requested article_id", () => {
     return request(app)
-      .get("/api/articles?topic=elephants")
-      .expect(400)
+      .get("/api/articles/1?query=comment_count")
+      .expect(200)
       .then(({ body }) => {
-        const { msg } = body;
-        expect(msg).toBe("bad request");
+        const { resultCount } = body;
+        expect(resultCount[0]).toMatchObject({
+          count: "11",
```

Comment:
We don't want this behaviour we want the `article` object to have an additional property of `comment_count`